mteb/tasks/PairClassification/OpusparcusPC.py

```diff
@@ -0,0 +1,68 @@
+from ...abstasks import AbsTaskPairClassification
+import datasets
+
+_LANGUAGES = ["de", "en", "fi", "fr", "ru", "sv"]
+
+
+class OpusparcusPC(AbsTaskPairClassification):
+    @property
+    def description(self):
+        return {
+            "name": "OpusparcusPC",
+            "hf_hub_name": "GEM/opusparcus",
+            "description": "Opusparcus is a paraphrase corpus for six European language: German, English, Finnish, French, Russian, and Swedish. The paraphrases consist of subtitles from movies and TV shows.",
+            "reference": "https://gem-benchmark.com/data_cards/opusparcus",
+            "category": "s2s",
+            "type": "PairClassification",
+            "eval_splits": ["test.full", "validation.full"],
+            "eval_langs": _LANGUAGES,
+            "main_score": "ap",
+            "revision": "9e9b1f8ef51616073f47f306f7f47dd91663f86a",
+        }
+
+    def __init__(self, langs=None, **kwargs):
+        super().__init__(**kwargs)
+        if type(langs) is list:
+            langs = [lang for lang in langs if lang in self.description["eval_langs"]]
+        if langs is not None and len(langs) > 0:
+            self.langs = langs  # TODO: case where user provides langs not in the dataset
+        else:
+            self.langs = self.description["eval_langs"]
+        self.is_multilingual = True
```

Comment:
Maybe I'm missing sth but can we remove this & have it inherit from MultilingualTask?

Reply:
Good point we made the change